train.py

```diff
@@ -291,6 +291,9 @@ def main(args):
     args = parse_args()
     hostname = socket.gethostname()
 
+    if args.bind_numa:
```

Comment:
Try not to modify train.py, we should strive to make our modifications effective without requiring users to modify the interface code.